### PR TITLE
install jQuery.mmenu using composer, fixed #3159

### LINKED ISSUE #3159: Include mmenu using composer
* Remove installation from admin module
* Install using composer instead

https://github.com/FrDH/jQuery.mmenu/blob/master/composer.json
After #3155 has been merged.

### PR DESCRIPTION
**Pull request by Guite**
| Q                 | A
| ----------------- | ---
| Bug fix?          | no
| New feature?      | no
| BC breaks?        | no
| Deprecations?     | no
| Fixed tickets     | #3159 
| Refs tickets      | -
| License           | MIT
| Changelog updated | yes

### CODE REVIEW

File: CHANGELOG-1.4.md [hunk 1]
```diff
@@ -34,23 +34,25 @@ CHANGELOG - ZIKULA 1.4.x
     - Added validation of extension composer files (#2669, #3151)
 
  - Vendor updates:
+    - assetic updated from 1.3.2 to 1.4.0
```
Comment (craigh) [inline CHANGELOG-1.4.md]: no - we removed ascetic completely

Comment (Guite) [inline CHANGELOG-1.4.md]: it is obviously still included transitively

File: CHANGELOG-1.4.md [hunk 2]
```diff
@@ -34,23 +34,25 @@ CHANGELOG - ZIKULA 1.4.x
...
     - symfony updated from 2.8.9 to 2.8.13
-    - symfony service definition validator updates from 1.2.5 to 1.2.6
+    - symfony service definition validator updated from 1.2.5 to 1.2.6
     - twig updated from 1.24.1 to 1.27.0
-    - knp-menu installed at version 2.2.0
-    - knp-menu-bundle installed at version 2.1.3
-    - liip/imagine-bundle installed at 1.6.0
-    - font-awesome updated from 4.6.3 to 4.7.0
-    - added justinrainbow/json-schema 4.0.1
-    - jstree updated from 3.3.2 to 3.3.3
```
Comment (craigh) [inline CHANGELOG-1.4.md]: why move all these around?

Comment (Guite) [inline CHANGELOG-1.4.md]: resorted them alphabetically for easier readability (like we did also for 1.4.3)

File: CHANGELOG-1.4.md (hunk 1 above)
Comment (craigh) [inline CHANGELOG-1.4.md]: I don't think so. how so? It should be prevented.

Comment (Guite) [inline CHANGELOG-1.4.md]: It is required by `robloach/component-installer`. Just look in `composer.lock`.

**Pull request merged**